src/app/helpers/functions_helper.rb

```diff
@@ -7,4 +10,63 @@ def render_code(ruby)
     sanitized = sanitize(formatter.format(lexer.lex(ruby)), tags: %w[div span], attributes: %w[style])
     tag.pre sanitized
   end
+
+  def to_html(str)
```

Comment:
[R] to match the other method name `render_code` that renders code snippet, this is similar but renders markdown, so the name should match
```suggestion
  def render_markdown(str)
```